crates/holochain/src/conductor/conductor.rs

```diff
@@ -1683,11 +1694,12 @@ mod builder {
             p2p_evt: holochain_p2p::event::HolochainP2pEventReceiver,
             post_commit_receiver: tokio::sync::mpsc::Receiver<PostCommitArgs>,
         ) -> ConductorResult<ConductorHandle> {
-            tokio::task::spawn(p2p_event_task(p2p_evt, handle.clone()));
-
-            let _ = handle
+            handle
                 .clone()
-                .start_scheduler(holochain_zome_types::schedule::SCHEDULER_INTERVAL);
+                .start_scheduler(holochain_zome_types::schedule::SCHEDULER_INTERVAL)
```

Comment:
I'm not 100% sure "burst" mode is good for the intervals if we're moving to 100ms... but I don't know that it's not good either, so I guess let's go with it until/unless there is a problem, right? : )

Reply:
@neonphog exactly, this parameter is so loose it seems like it could be almost anything, and actually i think probably one day it will be config somehow